### PR TITLE
Wraps api model data in the model name

### PR DESCRIPTION
**Pull request by earobinson**
- Show, Create, Delete, and Update use the singular name
- ShowAll uses the plural name

### CODE REVIEW

File: lib/utilities.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+const pluralizeImport = require('pluralize');
```
Comment (azharkhan) [inline lib/utilities.js]: `pluralizeLibrary` is also alright. but I get what you are doing here. was wondering why this name was changed, but seeing the exposed function makes sense.

Comment (earobinson) [reply lib/utilities.js]: ya its a little ugly, totally open to other options, this was the best I could think of.

File: lib/utilities.spec/utilities.singularize.spec.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    });
+  });
+
+  describe('complex pluralization\'s', () => {
+    it('must not singularize goose', () => {
+      expect(singularize('goose')).to.equal('goose');
+    });
+
+    it('must singularize geese', () => {
+      expect(singularize('geese')).to.equal('goose');
```
Comment (azharkhan) [inline lib/utilities.spec/utilities.singularize.spec.js]: love these tests.

**Review by azharkhan — APPROVED** [review]
changes look good to me.

**Review by Abhijith1995 — APPROVED** [review]

**Pull request merged**